packages/vite-plugin-qrcode/tsup.config.js

```diff
@@ -3,5 +3,6 @@ export const tsup = {
 	entryPoints: ['src/index.ts'],
 	format: ['esm', 'cjs'],
 	splitting: false,
-	clean: true
+	clean: true,
+	target: 'node14'
```

Comment:
use min node version as target
```suggestion
	target: 'node14.13.1'
```